src/input_adaptors/tables.py

```diff
@@ -0,0 +1,29 @@
+import json
+import os
+from input_adaptors.index import *
+from commands.keys import Keys
+
+
+class Tables:
+    def __init__(self, table_object):
+        self.table_object = table_object
+        self.name = table_object[Keys.NAME]
+        self.columns = table_object[Keys.COLUMNS]
+        self.primary_key = table_object[Keys.PRIMARY_KEY]
+        self.index = table_object[Keys.INDEX_KEYS]
+
+    def create_table(self, path):
+        self.validate(self)
+        path = os.path.join(path, self.name)
+        os.makedirs(path, exist_ok=True)
+        with open(os.path.join(path, "{}_schema.json".format(self.name)), 'w') as file:
+            json.dump(self.table_object, file)
+        for index in self.index:
+            index = Index(index)
+            index.validate(self.index)
```

Comment:
No need to call `index.validate` here? It should be called inside index constructor.